process/sandbox-annual-review.md

```diff
@@ -0,0 +1,43 @@
+# Sandbox annual review 
+
+Sandbox projects are subject to an annual review by the TOC. This is intended to be a lightweight process to ensure that projects are on track, and getting the support they need.
+
+## How to file your annual review
+
+CNCF staff will notify the project maintainers when the project review is due. 
+
+Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [cncf/toc/reviews](https://github.com/cncf/toc/tree/master/reviews). 
+
+* Raise a PR titled *[Project name] Annual Review*
+* The PR should include a file called `annual-<project name>-<year>.md` (for example, `annual-amazingproj-2019.md`) with the contents described below
```

Comment:
I would recommend doing a file name like:

```
2019-amazingproj-annual.md
```

In this file order is significant to the current round of reviews for the current year. Let's say that you want to open all of them in an editor

```
code 2019*.md
```